docs/outlook/append-on-send.md

```diff
@@ -104,33 +131,38 @@ For this scenario, instead of running the `action` function on choosing the **Pe
           <bt:ShortStrings>
             <bt:String id="GroupLabel" DefaultValue="Contoso Add-in"/>
             <bt:String id="TaskpaneButton.Label" DefaultValue="Show Taskpane"/>
-            <bt:String id="ActionButton.Label" DefaultValue="Perform an action"/>
+            <bt:String id="PrependButton.Label" DefaultValue="Prepend header on send"/>
+            <bt:String id="AppendButton.Label" DefaultValue="Append disclaimer on send"/>
           </bt:ShortStrings>
           <bt:LongStrings>
             <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Opens a pane displaying all available properties."/>
-            <bt:String id="ActionButton.Tooltip" DefaultValue="Perform an action when clicked."/>
+            <bt:String id="PrependButton.Tooltip" DefaultValue="Prepend the Contoso header on send."/>
+            <bt:String id="AppendButton.Tooltip" DefaultValue="Append the Contoso disclaimer on send."/>
           </bt:LongStrings>
         </Resources>
+        <!-- Configures the prepend-on-send and append-on-send features. The same value, AppendOnSend, is used for both features. -->
         <ExtendedPermissions>
           <ExtendedPermission>AppendOnSend</ExtendedPermission>
         </ExtendedPermissions>
       </VersionOverrides>
     </VersionOverrides>
     ```
 
+1. Save your changes.
+
 # [Teams Manifest (developer preview)](#tab/jsonmanifest)
 
 > [!IMPORTANT]
-> Append-on-send isn't yet supported for the [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md). This tab is for future use.
+> Append-on-send and prepend-on-send aren't yet supported for the [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md). This tab is for future use.
```

Comment:
I noticed you've been ordering prepend then append so thought this would keep it consistent
```suggestion
> Prepend-on-send and append-on-send features aren't yet supported for the [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md). This tab is for future use.
```